activerecord/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Rails 7.0.0.rc1 (December 06, 2021) ##
 
+*   Move database and shard selection config options to a generator.
+
+    Rather than generating the config options in the production.rb when applications are created, applications can now run a generator to create an initializer and uncomment / update options as needed. All multi-db configuration can be imlpemented in this initializer.
+
+    *Eileen M. Uchitelle*
```

Comment:
This change didn't make it in before v7.0.0.rc1 was released, so I don't think it's correct to list this under the `Rails 7.0.0.rc1 heading`. I've opened https://github.com/rails/rails/pull/43834 to move it above that heading.